etc/firejail.config

```diff
@@ -8,6 +8,9 @@
 # Disable U2F in browsers, default enabled.
 # browser-disable-u2f yes
 
+# Allow DRM execution in browsers, default disabled.
+# browser-allow-drm no
```

Comment:
also these two browser-* options are out of order (my fault)
but they are hard to move now because it might break user configs? or can they just be moved?

Reply:
I don't think moving them break anything. Distro package mangers usually don't overwrite config files automatically and even if they did then changing a comma will have same effect as rewriting whole thing from scratch. They're commented out by default so in case users would miss them while merging configs then they won't override any old settings.